src/Commands/Connection/InfoCommand.php

```diff
@@ -21,67 +21,48 @@ class InfoCommand extends TerminusCommand
      *
      * @command connection:info
      *
+     * @field-labels
+     *   sftp_username: SFTP Username
+     *   sftp_host: SFTP Host
+     *   sftp_password: SFTP Password
+     *   sftp_url: SFTP URL
+     *   sftp_command: SFTP Command
+     *   git_username: Git Username
+     *   git_host: Git Host
+     *   git_port: Git Port
+     *   git_url: Git URL
+     *   git_command: Git Command
+     *   mysql_host: Mysql Host
+     *   mysql_username: Mysql Username
+     *   mysql_password: Mysql Password
+     *   mysql_port: Mysql Port
+     *   mysql_database: Mysql Database
+     *   mysql_url: Mysql URL
+     *   mysql_command: Mysql Command
```

Comment:
Actually I discussed the labels with @ari-gold and we came to the conclusion that it was better not to relable the parameters in the output as you don't have any programattic way of mapping 'MySQL Command' => `--filter=mysql_command` it would only be implied through reading the help output and assuming a pattern.